a1/src/main/java/ro/utcn/spet/a1/model/Question.java

```diff
@@ -0,0 +1,45 @@
+package ro.utcn.spet.a1.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+//@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Question {
+   // @Id
+    //@GeneratedValue(strategy= GenerationType.IDENTITY)
```

Comment:
Having commented code is usually not a good sign :) That is why we use git --> if you delete code from a previous commit, you can still find it on git by inspecting the file's history.